ovirt_imageio/client/_options.py

```diff
@@ -45,13 +45,34 @@ def __repr__(self):
 log_level = Choices("log_level", ("debug", "info", "warning", "error"))
 
 
+class StoreFalseOrNoneAction(argparse.Action):
```

Comment:
StoreFalseOrNone is good enough. The Action thing come from the super class, and when
use we always use:

    action=StoreFalseOrNone(...)

So it is also clear.